protocol/chat.go

```diff
@@ -27,6 +27,13 @@ var chatColors = []string{
 	"#d37ef4", // purple
 }
 
+const (
+	// IncreaseUnviewedMessagesCountTimeout
+	// this timeout indicates how long the time between received messages should be
+	// for a new message to increase the unviewed messages counter
+	IncreaseUnviewedMessagesCountTimeout = 1000 * 60 * 2
```

Comment:
Maybe better to add the unit to the name, like `...CountTimeoutMs` to avoid any confusion when doing math.